### PR TITLE
Document how to ask engineers for overtime

### PR DESCRIPTION
**Pull request by richardTowers**
NOTE: this is a work in progress. I'm raising a PR for discussion. If people agree this process would be better than what we have now, then we'll adopt it and see how it works out. If it doesn't work well then we'll change it.

✨  [Rendered version](https://govuk-developer-docs-pr-2413.cloudapps.digital/manual/ask-for-technical-support-out-of-hours.html) 👀

### CODE REVIEW

**Comment by issyl0** [comment]
👏  This looks excellent and good to formalise. One question though: how do you/the teams decide who does overtime? Is it voluntary, or do you always pick certain people, or is it skills-based? Should it be spread evenly throughout the team members, if there are multiple occurrences, so that it doesn't unfairly burden one person, or so that other people can have the experience?

**Comment by richardTowers** [comment]
Thanks @issyl0!

I don't know what the process will be for deciding who does overtime yet. Hopefully we can come up with something that's fair though - all of your points are good.

I think I'd like to leave that bit of the process undocumented for now, because it's so unclear at the moment. We'll do our best to make any ad hoc assignments fair in the short term, and hopefully we can start to formalise things as we go.

File: source/manual/ask-for-technical-support-out-of-hours.html.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+work out of hours.
+
+## Work out what needs to be done
+
+First of all, make sure you understand the user need that's being addressed by the change.
+
+Talk to [the team that owns the application](/apps/by-team.html) you need to
+change to confirm that this is something that needs to be done by engineers out of hours.
+
+## Give us as much notice as possible
```
Comment (benthorner) [inline source/manual/ask-for-technical-support-out-of-hours.html.md:32]:
```suggestion
## Give developers as much notice as possible
```

File: source/manual/ask-for-technical-support-out-of-hours.html.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+We'll keep a close eye on the channel during working hours.
+
+You should be able to explain:
+
+* what you need
+* why it's important
+* why it needs to happen out of hours
+* why it needs an engineer to make the change
+
+You should let us know who the main point of contact should be from a
```
Comment (benthorner) [inline source/manual/ask-for-technical-support-out-of-hours.html.md:56]:
```suggestion
You should let the developer(s) know who the main point of contact should be from a
```

File: source/manual/ask-for-technical-support-out-of-hours.html.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+non-technical point of view.
+
+Let us know who you've worked with so far (for example if you've already spoken
```
Comment (benthorner) [inline source/manual/ask-for-technical-support-out-of-hours.html.md:59]:
```suggestion
Let the developer(s) know who you've worked with so far (for example if you've already spoken
```

**Comment by richardTowers** [comment]
> Thanks for raising this @richardTowers. I was thinking of approving, but got stuck with some of the wording: some of the uses of "us" and "we" are unclear, in that they seem to refer to a scoped group of developers, yet this may be read by any developer. I've made a few suggestions, but got stuck when I hit [the bit about "what we'll do"](https://github.com/alphagov/govuk-developer-docs/pull/2413/files#diff-0da9c7b069345c8b3a318a8102497f68R66).

I was actually thinking of "us" as a group of delivery managers and senior tech people who could have some oversight of who's being asked to do what.

This PR was raised before I knew about the other things being done to plan for overtime work, which have progressed a lot in the meantime. It probably needs a bit of a rethink in that light, so I'm going to close it for now, hopefully to be reopened soon once we've worked out some of the details around our current approach.

**Pull request closed without merging**